src/components/checkboxes/_checkbox-macro.njk

```diff
@@ -7,6 +7,7 @@
             id="{{ params.id }}"
             class="ons-checkbox__input ons-js-checkbox {{ params.inputClasses }}"
             value="{{ params.value }}"
+            {%if params.disabled %}disabled aria-disabled="true"{%endif %}
```

Comment:
We probably want to be a bit more verbose here and check for the value of `disabled`. Jinja2 in our python services can sometimes throw a wobbly when we don't define variables in specific ways. So maybe something like...

`{% if params.disabled is defined and params.disabled == true %}`

Reply:
👍